vector/src/main/java/im/vector/app/features/home/room/detail/timeline/helper/TimelineDisplayableEvents.kt

```diff
@@ -56,7 +56,8 @@ object TimelineDisplayableEvents {
 }
 
 fun TimelineEvent.canBeMerged(): Boolean {
-    return root.getClearType() == EventType.STATE_ROOM_MEMBER
+    return root.getClearType() == EventType.STATE_ROOM_MEMBER ||
+            root.getClearType() == EventType.STATE_ROOM_SERVER_ACL
```

Comment:
I am wondering if room member events could be merged with ACL events, which would be wrong, but your screenshots seems to show that this is not happening. Perfect!

Reply:
Yes, it is not possible. For what I understood, in the code, there is a grouping mechanism by same event type and same day.